cms/articles/tests/test_models.py

```diff
@@ -251,8 +252,125 @@ def test_breadcrumb_doesnt_containt_series_url(self):
             html=True,
         )
 
-    @override_settings(IS_EXTERNAL_ENV=True)
-    def test_load_in_external_env(self):
-        """Test the page loads in external env."""
+    def test_pagination_is_not_shown(self):
         response = self.client.get(self.basic_page_url)
-        self.assertEqual(response.status_code, 200)
+        expected = 'class="ons-pagination__link"'
+        self.assertNotContains(response, expected)
+
+
+class PreviousReleasesWithoutPaginationTestCase(TestCase):
+    # PREVIOUS_RELEASES_PER_PAGE is default value 10
+    @classmethod
+    def setUpTestData(cls):
+        cls.article_series = ArticleSeriesPageFactory(title="Article Series")
+        cls.articles = StatisticalArticlePageFactory.create_batch(9, parent=cls.article_series)
+        cls.previous_releases_url = cls.article_series.url + cls.article_series.reverse_subpage("previous_releases")
+
+    def test_breadcrumb_does_containt_series_url(self):
+        response = self.client.get(self.previous_releases_url)
+        parent_page = self.article_series.get_parent()
+        # confirm that current breadcrumb is there abd that current breadcrumb points to the parent page
+        # TODO currently this test points to the article page not to the series page
+        self.assertContains(
+            response,
+            f'<a class="ons-breadcrumbs__link" href="{parent_page.url}">{parent_page.title}</a>',
+            html=True,
+        )
+
```

Comment:
needs a
```python
        self.assertNotContains(
            response,
            f'<a class="ons-breadcrumbs__link" href="{self.article_series.url}">{self.article_series.title}</a>',
            html=True,
        )
```

to actually test that the breadcrumb doesn't have the series URL.

Reply:
I agree with your comment but there is a bug in the breadcrumb code and this will give a failing test hence the TODO comment have added this to the comment

Reply:
Thanks Dan!